docs/siem/detections/api/lists/api-import-list-items.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[lists-api-import-list-items]]
+=== Import list items
+
+Imports a list of items from a `.txt` file.
```

Comment:
For the UI, it will import anything from a `.txt` or `.csv` file. For the API on the backend, it can technically import form any file type. At the very least saying `.txt` or `.csv` would probably be the best but feel free to add any other words about how the API can actually do more and import from any file if you think that is important enough to call out.

Reply:
@FrankHassanabad - added `csv` files. For other file types, what is used as the delimiter between values?